Providing more detailed information for transactions grid, data, details, and Childs (#1198)
magento/merchdocs: src/sales/transactions.md

title: Transactions
---

The _Transactions_ grid lists all payment activity that has taken place between your store and a payment system. Each record provides access to more detailed information.

## View transactions

On the _Admin_ sidebar, go to **Sales** > _Operations_ > **Transactions**.

![]({% link images/images/sales-transactions.png %}){: .zoom}
_Transactions grid_

### Column descriptions

|Column|Description|
|--- |--- |

> |ID|A unique numeric identifier that is assigned for each transactional record.|

Reviewer: Please alphabetize things when possible: This table and the list of keys, even if it differs from the order in the admin.

Author: Thank you @dobooth !
Done, please approve :)

|Order ID|A unique identifier that is assigned when a customer places an order.|
|Transaction ID|A unique numeric identifier that is assigned when a transaction happens after a customer places an order.|
|Parent Transaction ID|The ID number of the parent transaction.|
|Payment Method| The payment method associated with a transaction.|

> |Transaction Type|Order, Authorization, Capture, Void, Refund|

Reviewer 2: ```suggestion
|Transaction Type|Type of transaction, which can be Order, Authorization, Capture, Void, or Refund.|
```

|Closed|Whether a transaction is closed or not.|
|Created|Time and date the transaction was created.|

> ## To view transaction details

Reviewer 2: We don't use infinitives in headings

```suggestion
## View transaction details
```

> 1. On the _Admin_ sidebar select **Sales**.
> 
> 1. Click **Transaction**.

Reviewer 2: ```suggestion
1. On the _Admin_ sidebar, go to **Sales** > **Transactions**.
```

1. Click the entry you want to view.

On the transaction details page, you can see the transaction detail and child transactions grid.

> ### Transaction Data

Reviewer 2: ```suggestion
### Transaction data
```

> This block includes information regarding the transaction, and links to the order page in the **Order ID** column.

Reviewer 2: ```suggestion
This section includes information regarding the transaction, and provides a link to the order page in the **Order ID** column.
```

|Column|Description|
|--- |--- |
|Transaction ID|The transaction ID number.|

> |Parent Transaction ID|A corresponding ID number of the parent transaction. Options: Numerical ID / N/A|

Reviewer 2: ```suggestion
|Parent Transaction ID|A corresponding ID number of the parent transaction, if applicable.|
```

> |Transaction Type|Type of transaction. Options: order, authorization, capture, void, refund.|

Reviewer 2: ```suggestion
|Transaction Type|Type of transaction, which can be Order, Authorization, Capture, Void, or Refund.|
```

> |Is Closed|Whether a transaction is closed or not. Options: Yes/No|

Reviewer 2: ```suggestion
|Is Closed|Whether a transaction is closed or not. |
```

|Created At|Time and date the transaction was created.|

### Child transactions

Child transactions are displayed in the grid after creating invoices for [orders]{% link sales/orders.md %}. This allows you to track transaction history, following a transaction hierarchy.

### Transaction Details

> This block includes the additional information for a given transaction. Information is displayed in the form of keys and values.
> 
> **Available Key**:

Reviewer 2: ```suggestion
This section includes the additional information for a given transaction. Information is displayed in the form of keys and values. The available keys are:
```
